### DISCUSSION TITLE
Aggressive cache not used for lazily loaded documents after first cache timeout

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#13967] [Q&A]
The following issue still impacts us:
https://issues.hibernatingrhinos.com/issue/RavenDB-16358
(report: https://groups.google.com/g/ravendb/c/a1-Zl147lcc/m/ekZod7J6GgAJ )

Is there any estimate about when this could be looked at?

Our use case is that in a web application, some data is aggressively cached for a period of time to save a database request on every request.

**Comment by arekpalinski** [comment]
Hi,

We will try to look into that next week.
